spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -624,7 +628,7 @@ function recvPacket(
 
     // all assertions passed (except sequence check), we can alter state
 
-    if (channel.order === ORDERED) {
+    if (channel.order === ORDERED || channel.order == ORDERED_ALLOW_TIMEOUT) {
       nextSequenceRecv = provableStore.get(nextSequenceRecvPath(packet.destPort, packet.destChannel))
```

Comment:
I don't understand. If packet sequence 5 times out, even if the channel doesn't close, I don't see any changes here that allow packet sequence 6 to be received

Reply:
Yes great catch, this makes the changes less trivial as we need to somehow communicate to destination chain the sequences that have been timed out